spec/integration/precompilation_spec.rb

```diff
@@ -43,4 +43,26 @@
     sprite_files = Dir.glob(File.join('public', 'assets', 'sprites', 'application-*.png'))
     expect(sprite_files.length).to eq(2)
   end
+
+  context 'a file is renamed' do
```

Comment:
I'm not sure I totally understand why you have to move the file twice to trigger the bug, but it makes this test feel a bit awkward, in that the description of the context ("a file is renamed") is not really representing what's happening in the before step.